### ISSUE TITLE
test 21_install.t hangs on Windows

### ISSUE THREAD

**Issue by chorny** [atrodo/App-MechaCPAN#2]
Considering nature of your distribution, it may be simpler to prevent test/installation by dying in Makefile.PL/Build.PL. See http://wiki.cpantesters.org/wiki/CPANAuthorNotes for how to do this.

**Comment by atrodo** [comment]
Thanks for the bug. I forgot to mention it, but I did work quite a bit on making MechaCPAN work on Win32. The issue was that Win32 can't run select on anything but sockets, so I had to rearrange code (and debug a lot of win32-isms) but it can now run and log output for commands ran on Win32. I've also added a command timeout so it should no longer ever hang and never progress on the testing.

The other end result of this is that I've disabled the perl part of MechaCPAN on Windows. Getting the code to build and install a local version of perl was too much of not-the-same that I've just disabled it.

In short, this should no longer be hanging on Windows.
